pkg/graph/resolvers/plan_data_exchange_approach.go

```diff
@@ -77,3 +79,8 @@ func PlanDataExchangeApproachUpdate(
 	retDataExchangeApproach, err := storage.PlanDataExchangeApproachUpdate(store, logger, existing)
 	return retDataExchangeApproach, err
 }
+
+// PlanDataExchangeApproachGetByModelPlanIDLoader calls a data loader to batch fetching a a plan data exchange object that corresponds to a model plan
+func PlanDataExchangeApproachGetByModelPlanIDLoader(ctx context.Context, modelPlanID uuid.UUID) (*models.PlanDataExchangeApproach, error) {
```

Comment:
With the introduction of this function, do we still need `PlanDataExchangeApproachGetByID`?
- [resolvers.PlanDataExchangeApproachGetByID](https://github.com/CMS-Enterprise/mint-app/blob/dc19ab115c2d626a3cf73b9cacc06b8d123ca138/pkg/graph/resolvers/plan_data_exchange_approach.go#L19)
- We should keep the `storage` method by the same name, since it's used in `resolvers.PlanDataExchangeApproachUpdate`